packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  storiesOf
+} from '@storybook/html';
```

Comment:
The above three lines can just be combined into one for now, as you have done below.